File: PHP/Cap05/c05ex01.php
<?php
// c05ex01.php

$I = 1;
while ($I <= 5) {
  printf("Entre um valor numerico: ");
  fflush(STDOUT);
  $N = intval(trim(fgets(STDIN)));

  $R = $N * 3;

  printf("Resultado = %d\n", $R);
  printf("\n");

  $I++;
}

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>

File: GO/Cap05/c05ex01.go
// c05ex01.go

package main
import "fmt"

func main() {
  var I, N, R int

  I = 1
  for (I <= 5) {
    fmt.Print("Entre um valor numerico: ")
    fmt.Scan(&N)

    R = N * 3

    fmt.Println("Resultado =", R)
    fmt.Println()

    I++
  }

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
